src/main/webapp/app/quiz/participate/multiple-choice-question/multiple-choice-question.component.ts

```diff
@@ -34,16 +35,23 @@ export class MultipleChoiceQuestionComponent implements OnInit, OnDestroy {
     @Input()
     fnOnSelection: any;
 
+    correctAnswers: number;
+    wrongAnswers: number;
+    chosenCorrectAnswerOption: number;
+    chosenWrongAnswerOption: number;
+
+    amountOfAnswerOptions: number;
+
     @Output()
     selectedAnswerOptionsChange = new EventEmitter();
 
     rendered: MultipleChoiceQuestion;
 
-    constructor(private artemisMarkdown: ArtemisMarkdown) {}
-
-    ngOnInit() {}
+    constructor(private artemisMarkdown: ArtemisMarkdown, private modalService: NgbModal) {}
```

Comment:
`private modalService: NgbModal` is unneeded

Reply:
I removed it